src/olympia/lib/settings_base.py

```diff
@@ -59,6 +59,23 @@ def path(*folders):
 
 DEBUG = env('DEBUG', default=False)
 
+# Do NOT provide a default value, this should be explicitly
+# set during the docker image build. If it is not set,
+# we want to raise an error.
+DOCKER_TARGET = env('DOCKER_TARGET')
+
+# "production" is a named docker stage corresponding to the production image.
+# when we build the production image, the stage to use is determined
+# via the "DOCKER_TARGET" variable which is also passed into the image.
+# So if the value is anything other than "production" we are in development mode.
+DEV_MODE = DOCKER_TARGET != 'production'
```

Comment:
Should we set that dynamically here, or be safer and set it to `False` here, but to `DOCKER_TARGET != 'production'` in settings.py ?

Reply:
/agree - this logic should be `settings.py` so it's always false on addons-dev, stage, and prod.

Reply:
Fair.. but if the DOCKER_TARGET in production is not production... we got problems.

Reply:
Updated